Sources/Hummingbird/Middleware/FileMiddleware.swift

```diff
@@ -94,8 +101,12 @@ public struct FileMiddleware<Context: RequestContext, Provider: FileProvider>: R
                 throw error
             }
 
+            guard request.method == .get || request.method == .head else {
+                throw error
```

Comment:
Do we need to throw an error, or can we forward to the next handler?

Reply:
We've already run the next handler. The file middleware only looks for files if the response is a 404